lms/djangoapps/discussion/tests/test_tasks.py

```diff
@@ -253,3 +254,41 @@ def test_subcomment_should_not_send_email(self):
 
     def test_second_comment_should_not_send_email(self):
         self.run_should_not_send_email_test(self.comment2)
+
+    @ddt.data((
+        {
+            'thread_id': 'dummy_discussion_id',
+            'thread_title': 'thread-title',
+            'thread_created_at': date.serialize(NOW),
+            'course_id': 'fake_course_edx',
+            'thread_author_id': 'a_fake_dude'
+        },
+        mock.patch('edx_ace.ace.send').start(),
```

Comment:
So, the only thing about this, is that you're not stopping the mock of ace.send after this, which could be problematic for any tests that run after this.  It might be clearer to do `with mock.patch('edx.ace.send')` in your `with` statement below, do your `setattr` stuff in front of the _track_notification_sent() call.